.idea/codeStyles/Project.xml

```diff
@@ -18,6 +18,9 @@
     <MarkdownNavigatorCodeStyleSettings>
       <option name="RIGHT_MARGIN" value="72" />
     </MarkdownNavigatorCodeStyleSettings>
+    <ScalaCodeStyleSettings>
+      <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
+    </ScalaCodeStyleSettings>
```

Comment:
Please remove changes of IDEA settings

Reply:
> Please remove changes of IDEA settings

Ok, I'll delete it after getting some response on the reply below.